src/qml/OverlayFeatureFormDrawer.qml

```diff
@@ -98,19 +97,8 @@ Drawer {
         focusstack.addFocusTaker( this )
     }
   }
+
   Component.onCompleted: {
       close()
   }
-
-  Keys.onReleased: {
-    if (event.key === Qt.Key_Back || event.key === Qt.Key_Escape) {
-      if( overlayFeatureForm.model.constraintsHardValid || qfieldSettings.autoSave ) {
-        overlayFeatureFormDrawer.close()
-      } else {
-        //block closing to fix constraints or cancel with button
-        displayToast( qsTr( "Constraints not valid" ) )
-      }
-      event.accepted = true
-    }
-  }
```

Comment:
is this ok to drop all of this?

Reply:
I think yes, it never gets triggered. The warning that targets this change says it all:

> Could not attach Keys property to: OverlayFeatureFormDrawer_QMLTYPE_58(0x55ef4b6bbaf0) is not an Item

which I interpret that the Drawer is not an Item, therefore no Keys property can be attached.

Reply:
ping @signedav

Reply:
Since I don't have a computer atm I cannot check it. This implementation has been made to close the drawer with back key (escape on computer) and has to be implemented on every item that could have the focus. Possibly this is not needed here. If it doesn't accept it as property then it's not, i guess...

Reply:
@signedav it is already in `overlayFeatureForm`, I am not sure why it was added to the `Drawer` element, most probably by mistake/copy-paste. I just checked both the `escape` key on desktop and back button on android, and they seem to work after my change.